compiler/test/dotty/tools/vulpix/ParallelTesting.scala

```diff
@@ -686,7 +685,9 @@ trait ParallelTesting extends RunnerOrchestration { self =>
 
     def getMissingExpectedErrors(errorMap: HashMap[String, Integer], reporterErrors: Iterator[MessageContainer]) = !reporterErrors.forall { error =>
       val key = if (error.pos.exists) {
-        val fileName = error.pos.source.file.toString
+        def toRelative(path: String): String =  // For some reason, absolute paths leak from the compiler itself...
```

Comment:
It is worth figuring out where they come from and why. Could or should they already be relative? This is a separate issue.

Reply:
Technically, both relative and absolute paths are correct ways to point to a file. If the compiler at some point converts relative paths to absolute ones, I don't think it is an issue.

Reply:
That can be an issue. For example `.tasty` files contain a path to the file which will be published. This path must be relative or any tool that consumes them will be lost.